src/extension.ts

```diff
@@ -403,6 +404,29 @@ export async function activate(context: vscode.ExtensionContext) {
     });
   });
 
+  registerIDFCommand("espIdf.fullClean", () => {
+    PreCheck.perform([openFolderCheck], async () => {
+      const buildDir = path.join(workspaceRoot.fsPath, "build");
+      const buildDirExists = await utils.dirExistPromise(buildDir);
+      if (!buildDirExists) {
+        Logger.infoNotify(`Directory ${buildDir} does not exists.`);
```

Comment:
Maybe something like this `There is no build directory to clean, exiting!`. Also maybe its better to use warn than info

Also can use `return` in same line